cpp/oneapi/dal/algo/pca/test/batch.cpp

```diff
@@ -47,6 +47,20 @@ class pca_batch_test : public te::algo_fixture {
                         std::int64_t component_count,
                         const te::table_id& data_table_id) {
         CAPTURE(component_count);
+        // {
+        //     const table y = data.get_table(data_table_id, 1, 1);
+
+        //     const auto ptr = dal::row_accessor<const float>{y}.pull().get_data();
+
+        //     std::cout << "batch" << std::endl;
+        //     for (size_t i = 0; i < y.get_row_count(); ++i) {
+        //         for (size_t j = 0; j < y.get_column_count(); ++j) {
+        //             std::cout << ptr[i * y.get_column_count() + j] << ' ';
+        //         }
+        //         std::cout << std::endl;
+        //     }
+        // }
```

Comment:
```suggestion
```
Please don't comment code. This code will be stored in GIT anyway so you'll be able to recover it in any time.